.github/std-unexploded.yml

```diff
@@ -106,3 +106,48 @@ jobs:
       - uses: divnix/std-action/run@main
         with: {ffBuildInstructions: true, remoteStore: "ssh-ng://eu.nixbuild.net"}
 
+  deploy-to-eu:
```

Comment:
Why is it region-specific?

Reply:
The kubecontext is acquired by the CI role from the region variable and is also only valid for that region, unfortunately.

It is subobtimal. I have investigated k8s region topology just a bit in an attempt to find better long term answers. Another intermediate one would be to simply parallelise GH actions jobs. In any case, this needs more investigation, still.

Reply:
The most straight forward way was to work with the `aws` cli without complicated wrapping / manipulation:

```yaml
      - name: Configure K8S Cluster Access
        shell: bash
        run: |
          echo "Assuming role '$(aws sts get-caller-identity)' in cluster 'lace-prod-eu-central-1'."
          aws eks update-kubeconfig --name "lace-prod-eu-central-1"
```

I think this  sets a good precedent of simplicity, at least for this iteration.

It is also relevant in combination with #865 which requires us to gather more practical experience with how this integration ends up working and in which way we want to present and map environments.

Reply:
I think this is can be resolved here?

Since this question will resurface as soon as we increase the coverage of environments one way or another, I'll abstain from a dedicated backlog, here.